docs/sdk/crashes/android.md

```diff
@@ -100,6 +124,11 @@ public boolean shouldProcess(ErrorReport report) {
      return true; // return true if the crash report should be processed, otherwise false.
 }
 ```
+```kotlin
+override fun shouldProcess(report: ErrorReport?): Boolean {
+	return true
```

Comment:
please fix indentation

Reply:
Weird, it is only indented by 4 spaces here...